zx.mjs

```diff
@@ -31,6 +32,10 @@ Object.assign(global, {
   os,
 })
 
+function urlJoin(...paths) {
+  return pathToFileURL(join(...paths))
```

Comment:
Because you wouldn't be able to run `node zx.mjs test.mjs` otherwise. Specifically, #28 was closed erronously -- nodejs still complains about `c:` not being a valid protocol.